drivers/rtaudio/audio_driver_rtaudio.cpp

```diff
@@ -32,8 +32,14 @@ const char* AudioDriverRtAudio::get_name() const {
 int AudioDriverRtAudio::callback( void *outputBuffer, void *inputBuffer, unsigned int nBufferFrames,
 	double streamTime, RtAudioStreamStatus status, void *userData ) {
 
-	if (status)
-		print_line("lost?");
+	if (status) {
+		if (status == RTAUDIO_INPUT_OVERFLOW) {
+			WARN_PRINT("RtAudio input overflow!");
+		}
+		if (status == RTAUDIO_OUTPUT_UNDERFLOW) {
```

Comment:
If I'm to be nit-picking, I'd say it should be either an `else if`, or both tests should use the `&` bitwise operator :)